src/components/LogIn/LogIn.js

```diff
@@ -45,11 +45,10 @@ export default function LogIn({ children }) {
             {generateErrorMessage(loginError)}
           </div>
         </div>
-        <main>{children}</main>
```

Comment:
This implementation was causing the whole app to be torn down and re-rendered after logging in.